### DISCUSSION TITLE
add information attributes to mesh

### DISCUSSION THREAD

**Discussion by nikita-petrashen** [cnr-isti-vclab/PyMeshLab#127] [Ideas] [answered]
It would be very convenient to access `Mesh.label` and `Mesh.fullName` from Python directly via class attributes. Or is it possible and I just didn't find the way to do this?

**Comment by nikita-petrashen** [comment]
Also the messages sent to `std::cout` are not displayed in Jupyter notebooks, here's a possible [way](https://pybind11.readthedocs.io/en/stable/advanced/pycpp/utilities.html#capturing-standard-output-from-ostream) of overcoming this

**Comment by alemuntoni** [comment, marked as answer]
Thanks for the suggestions, @nikita-petrashen. 
I was already working on adding label and fullname to meshes, but it is not easy at the moment, and lot of reorganization in the codebase of meshlab is needed first. When this will be done, it will be possible also to iterate over meshes into the meshset, for example. 
I hope to make this available in the next version of pymeshlab, but I cannot promise anything at the moment.
